src/journal/journald-server.c

```diff
@@ -1577,8 +1611,28 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
 }
 
 static int server_parse_config_file(Server *s) {
+        int r;
+
         assert(s);
 
+        if (s->namespace) {
+                const char *namespaced;
+
+                /* If we are running in namespace mode, load the namespace specific configuration file, and nothing else */
+                namespaced = strjoina(PKGSYSCONFDIR "/journald@", s->namespace, ".conf");
```

Comment:
I thought about this, but I ultimately figured this is not appropriate, since the settings in the configuration file are quite different for the namespaces and the host, since there's ReadKMsg= and such options. Options like that make sense for the main namespace but very little for the auxiliary namespaces. Hence incorporating the host configuration by default seemed very wrong to me. 

I mean, there are two conflicting goals here: the goal to make namespaces dead cheap and easy to use (which means zero preparation such as copying in a config file for each namespace before using it) and the goal of having a single source of default configuration that affects all instances. But these two items conflict, since the configuration for the instances that are not the main one really ought to be different than the main one...

So for now I opted to just not pull in the config options from the main config, and keep things separate. My thinking was that it is safer to enable the inheriting from the main config later than to drop it again eventuallly...

But yeah, I implicitly assumed we should reuse the config of the main journal here, too, but it conflicts with the intended behaviour otherwise.

Opinions? Ideas about this?